scipy/stats/_qmc.py

```diff
@@ -389,7 +416,7 @@ def n_primes(n):
         List of primes.
 
     """
-    primes = [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59,
+    primes = np.array([2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59,
```

Comment:
I had to add `np.array` at the front because n_primes is returning a `np.ndarray`